python/src/etos_api/library/validator.py

```diff
@@ -176,3 +177,51 @@ async def validate(self, test_suite):
                 assert (
                     await docker.digest(test_runner) is not None
                 ), f"Test runner {test_runner} not found"
+
+
+class ArtifactValidator:
```

Comment:
Isnt this validation already being made by the fact that we use StartEtosTestrun & StartEtos (or whatever they are called) pydantic models as input to our FastAPI functions?

I think the real problem with the ETOS API validation is that the client does not understand the response or we need to update the response code from the API (I think it is 422 Unprocessable Entity)

Reply:
Im happy with this change, but @fredjn will have to approve it